style.css

```diff
@@ -0,0 +1,322 @@
+body {
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+/* Header */
+header {
+  background-color: #f1efe9;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.logo {
+  width: 25%;
+  max-width: 142px;
+  border-radius: 10%;
+  margin: 0 2px;
+}
+
+header h1 {
+  font-family: 'Indie Flower', cursive;
+  font-size: 40px; 
+  margin: 20px 0;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: #000;
+  text-decoration: none;
+  margin-left: 12px;
+  
+}
+
+nav a :hover {
+  color: #44bfed;
+}
+
+.hamburger {
+  display: flex; 
+  width: 25px;
+  height: 25px; 
+  margin-right: 2px;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div {
+  width: 25px;
+  height: 5px;
+  background: #000;
+}
+
+/* Main Content */
+
+.container .main-news .main-flex-box {
+  border-bottom: 2px solid #44bfed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 50px;
+}
```

Comment:
When written without commas this styling will only affect the `.main-flex-box`. If you want it to affect all three classes it needs to written like this: 
```
.container, .main-news, .main-flex-box {
...
}
```
If you only wanted to affect the inner most container, since you have a class name for it, writing it like this would be enough: 
```
.main-flex-box {
...
}
```